packages/mgt-chat/src/statefulClient/GraphNotificationUserClient.ts

```diff
@@ -238,7 +238,8 @@ export class GraphNotificationUserClient {
     }
   }
 
-  private trySwitchToDisconnected() {
+  private trySwitchToDisconnected(ignoreIfUndefined = false) {
```

Comment:
I'm curious why `ignoreIfUndefined` is exposed as a parameter when all the usages just pass `true` anyway